sample/readme.md

```diff
@@ -0,0 +1,17 @@
+# Sample Data and figures
+
+## Machine configurations
+
+The following machine configurations were used to obtained the data and figures stored within the repository. The folder format uses the following convention;
+
+```
+[gpu short name]-[CUDA version]-[Driver version]\[FLAMEGPU2 Release tag]-[gpu short name]-[CUDA version]-beltsoff
```

Comment:
This does not match the actual pattern used? There is no CUDA version in the higher directory.

Reply:
corrected in the readme